ui/raidboss/data/07-dt/trial/byakko-un.ts

```diff
@@ -0,0 +1,421 @@
+// This file was autogenerated from running npm run sync-files.
+// DO NOT EDIT THIS FILE DIRECTLY.
+// Edit the source file below and then run `npm run sync-files`
+// Source: ui/raidboss/data/04-sb/trial/byakko-ex.ts
+
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  roarCount?: number;
+  stakeCount?: number;
+}
+
+// Byakko Unreal
+const triggerSet: TriggerSet<Data> = {
+  id: 'TheJadeStoaUnreal',
+  zoneId: ZoneId.TheJadeStoaUnreal,
+  timelineFile: 'byakko-un.txt',
```

Comment:
```suggestion
  timelineTriggers: [
    {
      id: 'ByakkoUn Fell Swoop',
      regex: /Fell Swoop/,
      beforeSeconds: 5,
      response: Responses.bigAoe('alert'),
    },
  ],
```
This is the big raidwide after the intermission phase; it doesn't have a castbar to trigger off of normally.